packages/manager/modules/carrier-sip/src/dashboard/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,22 @@
+{
+  "carrier_sip_dashboard_header_dashboard": "Tableau de bord",
+  "carrier_sip_dashboard_header_endpoints": "SIP Endpoints",
+  "carrier_sip_dashboard_header_cdr": "CDR",
+  "carrier_sip_dashboard_general_information": "Informations générales",
+  "carrier_sip_dashboard_general_information_name": "Nom du service",
+  "carrier_sip_dashboard_general_information_description": "Description",
+  "carrier_sip_dashboard_general_information_max_calls_second": "Nombre de call/sec max",
```

Comment:
```suggestion
  "carrier_sip_dashboard_general_information_max_calls_second": "Nombre d'appels/sec max",
```
?

Reply:
We cannot change this label. It has been provided and validated as is in the mockup.

Reply:
It's not because is validated by CX or Content that it's sensical 😄